### ISSUE TITLE
material Snackbar only showed once when Koalaplot is added to project

### ISSUE THREAD

**Issue by tahaak67** [KoalaPlot/koalaplot-core#36]
First of all thanks for the great work on Koalaplot, i love it!.

i've had this wiered behaviour with snackbar when koalaplot library is added to the project, the snackbar showes only once and cannot be shown the second time calling 'showsnackbar' function.

**effected targets:**

- iOS (Material 2 & 3)
- Desktop (Material 3)

**here are the steps to reproduce the issue:**

1. Create a new compose multiplatform project using: https://kmp.jetbrains.com/
2. Replace the code in App.kt in 'commonmain'
``` kotlin
import androidx.compose.foundation.layout.Column
import androidx.compose.foundation.layout.fillMaxWidth
import androidx.compose.material.*
import androidx.compose.runtime.Composable
import androidx.compose.runtime.getValue
import androidx.compose.runtime.mutableStateOf
import androidx.compose.runtime.remember
import androidx.compose.runtime.rememberCoroutineScope
import androidx.compose.runtime.setValue
import androidx.compose.ui.Alignment
import androidx.compose.ui.Modifier
import kotlinx.coroutines.launch

@Composable
fun App() {
    MaterialTheme {
        val snackbarHostState = remember { SnackbarHostState() }
        val scope = rememberCoroutineScope()
        var x by remember { mutableStateOf(1) }
        Scaffold(modifier = Modifier,
            snackbarHost = { SnackbarHost(snackbarHostState) }) {
            Column(Modifier.fillMaxWidth(), horizontalAlignment = Alignment.CenterHorizontally) {
                Button(onClick = {
                    scope.launch {
                        snackbarHostState.showSnackbar("Snackbar shown this many times: $x")
                        x++
                    }
                }) {
                    Text("Show snackbar!")
                }
            }
        }
    }
}
```
4. Add Koalaplot library to build.gradle.kts and sync
5. If you run the app on one of the effected targets snackbar will only be shown the first time.

if you remove the koalaplot library and sync then run the snackbar will be shown normally.

**Comment by gsteckman** [comment]
I tried this on Desktop (I don't have any Apply devices to try iOS), and I don't see the behavior you describe.

**Comment by tahaak67** [comment]
> I tried this on Desktop (I don't have any Apply devices to try iOS), and I don't see the behavior you describe.

did you try Desktop with material 3?

**Comment by gsteckman** [comment]
I just tried with that as well, and it will popup the snackbar more than once. If you click the button multiple times quickly it doesn't increment the count correctly but if you wait for the snackbar to go away, then click the button again, it shows with the correct total count of clicks. But I don't think that is the issue you are reporting (that behavior was the same with or without Koala Plot).

I'm using Desktop on Windows 10. Are you on a Mac or Linux? Could be a platform dependent issue?

**Comment by tahaak67** [comment]
This is a known issue in compose multiplatform `1.6.0-alpha01` (i guess Koala plot uses this version) this should be fixed by upgrading to 1.6.0-beta01 as stated [here](https://stackoverflow.com/a/77879009/4906512).
closing this as its not an issue with Koalaplot

**Comment by gsteckman** [comment]
Thanks for tracking that down. I made a new release 0.5.2 that depends on 1.6.0-beta01.
